adal/build.gradle

```diff
@@ -141,7 +141,7 @@ dependencies {
 
     // 'local' flavor dependencies
     localApi(project(":common")) {
```

Comment:
>localApi [](http://example.com/codeflow?start=4&length=8)

Why this is only needed for local variant?

Reply:
because dist pulls from Maven and that doesn't have common4j yet...we will have to do it for dist as well when we publish common4j...added todo as well: https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/1601/commits/2a2db290221a8533307e47091b3ec7fea3217e6c

Reply:
I am still not clear on, why this only matters for local variant. as long as the verison of :common is same, shouldnt it be varian agnostic. unless we are doing something in common project to be variant specific. are we?